docs/setup/navqplus.md

```diff
@@ -9,14 +9,14 @@
 
 ## Step-by-step
  
-1. Download the [pre-built Ubuntu 20.04 with ROS2 Galactic image](https://github.com/rudislabs/navqplus-create3-images/releases), specifically designed for use with the Create® 3.[^1][^3]
+1. Download the [pre-built latest Ubuntu 22.04 with ROS2 Humble and CycloneDDS image](https://github.com/rudislabs/navqplus-create3-images/releases/latest), specifically designed for use with the Create® 3.[^1][^3]
```

Comment:
Is this Humble image meant to be used with a Galactic Create 3 robot?
I think that we should specify:
 - which versions of Create 3 this is compatible with (galactic or humble)
 - if using Create 3 with ROS 2 Humble, the robot will be using Fast-DDS by default so we should mention that it should be changed (either on robot or on NavQPlus)

Reply:
I agree -- let's just be explicit about the above -- otherwise this PR looks good to me.

Reply:
@alsora good catch, I will add a portion about changing the Create 3 RMW. We intentionally baked in the image default to use CycloneDDS with an updated config file for it as well as iceoryx support. This allows us to go directly from zenoh-pico with CDR into CycloneDDS/ROS2 using zenohd with the dds plugin built.